classes/submission/reviewRound/ReviewRound.php

```diff
@@ -250,6 +250,7 @@ public function determineStatus()
                     break;
 
                 case ReviewAssignment::REVIEW_ASSIGNMENT_STATUS_AWAITING_RESPONSE:
+                case ReviewAssignment::REVIEW_ASSIGNMENT_STATUS_REQUEST_RESEND:
```

Comment:
If a review round have all reviews sorted out (submitted and confirmed) and then a new reviewer added who has declined and then again re-requested, The review round show status as `ReviewRound::REVIEW_ROUND_STATUS_REVIEWS_COMPLETED ==> 9` but it show as `ReviewRound::REVIEW_ROUND_STATUS_PENDING_REVIEWS ==> 7` as the review has not yet responded to re-request .